release_notes/ocp_3_5_release_notes.adoc

```diff
@@ -0,0 +1,1691 @@
+Applications Across Environments] and
+xref:../dev_guide/application_lifecycle/new_app.adoc#dev-guide-new-app[Creating New Applications] for
+more information.
+
+[[ocp-35-web-console]]
+=== Web Console
+
+[[acp-35-run-and-deploy-on-ocp]]
+==== Run and Deploy on {product-title}
+
+In {product-title} 3.5, there is now a "Run on OpenShift" experience that allows you to
+provide external links in the web console to deploy templates.
+
+Use the URL pattern to select a template or image. You can customize to have it
+come from separate project. The end-user is prompted for the project.
+
+[[ocp-35-web-console-added-service-details]]
+==== Added Service Details
+
+There are now added service details on configuration, traffic, routes, and pods.
+
+There is a new section highlighting routes, service and target ports, host name,
+and TLS. There is also a section iterating pods and their status.
+
+.Service Details View
+image::ocp35-service-details.png[Service Details]
+
+[[ocp-35-web-console-configmap-create-list-detail]]
+==== ConfigMap: Create, List, Detail
+
+In {product-title} 3.5, there is now the ability to
+easily work with configuration data decoupled from the container image. You can:
+
+* Create new `ConfigMap`
+* List out existing `ConfigMaps`
+* Work with the configuration details.
+* Easily consume them from various other pages.
+
+.Create a ConfigMap
+image::ocp-35-create-configmap.png[Create a ConfigMap]
+
+.Add Config Files
+image::ocp-35-add-config-files.png[Add Config Files]
+
+
+[[ocp-35-web-console-show-build-failures]]
+==== Show Build Failures
+
+Users no longer have to search logs to gain a better understanding of why build
+failed. Individual build status messages are now updated with details that are
+available via the web console and the CLI.
+
+.Build Failures as Seen in the Web Console
+image::ocp35-build-failure.png[Build Failures in the Web Console]
+
+.Build Failures as Seen in the CLI
+image::ocp35-build-failure-cli.png[Build Failures in the CLI]
+
+[[ocp-35-web-console-statefulsets]]
+==== StatefulSets (Technology Preview)
+
+Custom resource listing and details pages for `StatefulSets` (formerly known as
+`PetSets`) is now available. Users can get details of all `StatefulSets`,
+including deployments and replica sets.
+
+See the
+xref:../architecture/infrastructure_components/web_console.adoc#web-console-statefulsets[Web
+Console] documentation for more information.
+
+
+[[ocp-35-notable-technical-changes]]
+== Notable Technical Changes
+
+{product-title} 3.5 introduces the following notable technical changes.
+
+[discrete]
+[[ocp-35-updated-infrastructure-components]]
+=== Updated Infrastructure Components
+
+{product-title} 3.5 is supported on RHEL 7.2 and 7.3 with the latest packages
+from Extras, including Docker 1.12. {product-title} 3.5 is not certified with
+older versions of Docker and RHEL 7.1 or earlier.
```

Comment:
We should mention Kubernetes has been updated to v1.5 and etcd to just 3.1. A la:

https://docs.openshift.com/container-platform/3.4/release_notes/ocp_3_4_release_notes.html#ocp-34-notable-technical-changes